modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -162,7 +188,15 @@
                 placement: 'bottom'
               }"
               type="subtle" :modifiers="['small', 'no-motion']"
-              @click="switchToPreviewMode"
+              @click="switchEditMode(false)"
+            />
+            <AposButton
+              v-if="editMode"
+              type="primary" label="Publish Changes"
+              :disabled="!(readyToSave && draftIsModified)"
```

Comment:
@boutell this seems like it would be the correct test to disable the Publish button but `readyToSave` seems kinda flaky.. One broken case is: make some changes, autosave, refresh, then `readyToSave` is no longer true.

This needs to be sorted before merge

Reply:
@stuartromanek I pushed up a new commit clarifying this and covering another case. Seems to work well now, but let me know your impression as well.